crates/config/src/signing.rs

```diff
@@ -0,0 +1,109 @@
+use figment::Figment;
+use figment::providers::{Format, Json, Toml};
+use serde::{Deserialize, Serialize};
+use serde_keys::Bn254SecretKey;
+use shellexpand::tilde;
+use std::num::NonZeroU16;
+use std::path::PathBuf;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct SecretKeyConfig {
+    pub node_id: NonZeroU16,
+    pub secret_key: Bn254SecretKey,
```

Comment:
This probably should be a more generic interface for the secret key, but probably worth opening an issue for that @azixus will need to fix in #58